Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'loofah', '>= 2.3.1'
 gem 'rack', '~> 2.0'
 gem 'ffi', '1.9.24'
 gem 'rails-controller-testing'
+gem 'flipper'
```

Comment:
Does rubocop have similar 'in alphabetical order' rules for requires?

Reply:
No rules about require order, that I could find.